source/rst/tools_and_techniques/orth_proj.rst

```diff
@@ -757,7 +757,7 @@ idempotence of :math:`M` and :math:`P` is that both are orthogonal
 projections. After a point is projected into a given subspace, applying
 the projection again makes no difference. (A point inside the subspace
 is not shifted by orthogonal projection onto that space because it is
-already the closest point in the subspace to itself.)
+already the closest point in the subspace to itself).
```

Comment:
Technically it was grammatically correct the old way, but it doesn't really matter. This is clearer.